pkg/os/testing/fake_os.go

```diff
@@ -252,3 +253,35 @@ func (f *FakeOS) Hostname() (string, error) {
 	}
 	return "", nil
 }
+
+type FakeOSManager struct {
```

Comment:
I think this type isn't necessary if the `statManager` would be a wrapper around the `OS` interface.